test/HostnameTest.php

```diff
@@ -506,6 +506,25 @@ public function testAdditionalUTF8TLDs()
         }
     }
 
+    public function testAdditionalPunycodedTLDs()
+    {
+        $validator = new Hostname(Hostname::ALLOW_ALL);
+        // Check UTF-8 TLD matching
+        $valuesExpected = [
```

Comment:
Only for ``testAdditionalPunycodedTLDs`` or also for ``testAdditionalUTF8TLDs`` (merged)?

Example dataProvider (valid hostnames):
```
public function validTLDHostnames()
{
    // @codingStandardsIgnoreStart
    return [
        'test123.онлайн' => 'test123.онлайн',
        'test123.онлайн' => 'test123.xn--80asehdb',
        'тест.рф'        => 'тест.рф',
        'тест.рф'        => 'xn--e1aybc.xn--p1ai',
        'туршилтын.мон'  => 'туршилтын.мон',
        'туршилтын.мон'  => 'xn--h1aggjjdd5b4a.xn--l1acc',
    ];
    // @codingStandardsIgnoreEnd
}
```